File: languages/python/basics/03_condicionais.py
"""
Objetivo: praticar estruturas condicionais.
Como executar: python .\languages\python\basics\03_condicionais.py
Conceitos treinados: if, elif, else, comparação.
Próximo exercício: 04_lacos.py
"""

def classificar_nota(nota: float) -> str:
    if nota >= 9:
        return "excelente"
    if nota >= 7:
        return "boa"
    if nota >= 5:
        return "regular"
    return "insuficiente"

for nota in [9.5, 7.2, 5.8, 3.9]:
    print(f"Nota {nota}: desempenho {classificar_nota(nota)}")

File: languages/java/basics/03_Condicionais.java
/*
Objetivo: praticar condicionais em Java.
Como executar:
  cd .\languages\java\basics
  javac 03_Condicionais.java
  java Condicionais
Conceitos treinados: if, else if, else.
Próximo exercício: 04_Lacos.java
*/

class Condicionais {
    public static void main(String[] args) {
        int nota = 8;

        if (nota >= 9) {
            System.out.println("Excelente.");
        } else if (nota >= 7) {
            System.out.println("Aprovado.");
        } else if (nota >= 5) {
            System.out.println("Recuperação.");
        } else {
            System.out.println("Reprovado.");
        }
    }
}
